core/server/models/index.js

```diff
@@ -1,61 +1,69 @@
-var _       = require('lodash'),
-    Promise = require('bluebird'),
-    requireTree   = require('../require-tree'),
+/**
+ * Dependencies
+ */
+
+var Promise = require('bluebird'),
+    _ = require('lodash'),
+
+    exports,
     models;
 
-models = {
-    excludeFiles: ['_messages', 'base', 'index.js'],
-
-    // ### init
-    // Scan all files in this directory and then require each one and cache
-    // the objects exported onto this `models` object so that every other
-    // module can safely access models without fear of introducing circular
-    // dependency issues.
-    // @returns {Promise}
-    init: function init() {
-        var self = this;
-
-        // One off inclusion of Base file.
-        self.Base = require('./base');
-
-        // Require all files in this directory
-        return requireTree.readAll(__dirname, {followSymlinks: false}).then(function then(modelFiles) {
-            // For each found file, excluding those we don't want,
-            // we will require it and cache it here.
-            _.each(modelFiles, function each(path, fileName) {
-                // Return early if this fileName is one of the ones we want
-                // to exclude.
-                if (_.contains(self.excludeFiles, fileName)) {
-                    return;
-                }
-
-                // Require the file.
-                var file = require(path);
-
-                // Cache its `export` object onto this object.
-                _.extend(self, file);
-            });
-
-            return;
-        });
-    },
-    // ### deleteAllContent
-    // Delete all content from the database (posts, tags, tags_posts)
-    deleteAllContent: function deleteAllContent() {
-        var self = this;
-
-        return self.Post.findAll().then(function then(posts) {
-            return Promise.all(_.map(posts.toJSON(), function mapper(post) {
-                return self.Post.destroy({id: post.id});
+/**
+ * Expose all models
+ */
+
+exports = module.exports;
+
+models = [
+    'accesstoken',
+    'app-field',
+    'app-setting',
+    'app',
+    'client-trusted-domain',
+    'client',
+    'permission',
+    'post',
+    'refreshtoken',
+    'role',
+    'settings',
+    'tag',
+    'user'
+];
+
+function init() {
+    exports.Base = require('./base');
+
+    models.forEach(function (name) {
+        _.extend(exports, require('./' + name));
+    });
+
+    return Promise.resolve();
+}
+
+/**
+ * TODO: move to some other place
+ */
+
+// ### deleteAllContent
+// Delete all content from the database (posts, tags, tags_posts)
+exports.deleteAllContent = function deleteAllContent() {
+    var self = exports;
+
+    return self.Post.findPage({limit: 'all'}).then(function then(data) {
```

Comment:
There is still a rebase error here, this code should look like this: https://github.com/TryGhost/Ghost/blob/master/core/server/models/index.js#L44